app/api/v2/handlers/health_api.py

```diff
@@ -29,7 +29,7 @@ async def get_health_info(self, request):
         mapping = {
             'application': 'Caldera',
             'version': app.get_version(),
-            'access': access[0].name,
+            'access': access[0].name if len(access) > 0 else None,  # 0 when not authenticated.
```

Comment:
Let's update this API endpoint so that it returns a 401 if not authenticated. That way we aren't revealing more information than we want to

Reply:
Done.

Reply:
Thank you!